LayoutTests/fast/shadow-dom/slot-display-contents-composed-tree-after-display-toggle.html

```
@@ -0,0 +1,35 @@
+<script>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)